src/wrapped/lib/raw_bindings.c

```diff
@@ -16,67 +16,99 @@
 #include "ctypes_cstubs_internals.h"
 
 CAMLprim value reason_skia_paint_set_color(
-value vPaint,
-int32_t color
-) {
-   sk_paint_t* pPaint = CTYPES_ADDR_OF_FATPTR(vPaint);
+    value vPaint,
+    int32_t color)
+{
+   sk_paint_t *pPaint = CTYPES_ADDR_OF_FATPTR(vPaint);
```

Comment:
There's a lot of noise from the auto-formatting here as well. It doesn't seem like these were consistenly formatted anyway though, so at least it doesn't break established conventions. I'll do a follow-up PR with clang-format but, again, it would be nice if you turned off global auto-formatting to avoid the noise this creates.